Sound/Tidal/Pattern.hs

```diff
@@ -327,7 +328,9 @@ density :: Pattern Time -> Pattern a -> Pattern a
 density = fast
 
 _density :: Time -> Pattern a -> Pattern a
-_density r p = withResultTime (/ r) $ withQueryTime (* r) p
+_density r p | r == 0 = silence
```

Comment:
could also be:

```
_density r p | r == 0 = silence
                     | r < 0 = rev $ _density (0-r) p
                     | otherwise = withResultTime (/ r) $ withQueryTime (* r) p
```

Reply:
Thanks very much, I think that's what I meant to do!